Backend/Remora.Discord.API.Abstractions/API/Rest/IDiscordRestChannelAPI.cs

```diff
@@ -94,11 +95,12 @@ Task<Result<IChannel>> ModifyChannelAsync
         Optional<int?> bitrate = default,
         Optional<int?> userLimit = default,
         Optional<IReadOnlyList<IPartialPermissionOverwrite>?> permissionOverwrites = default,
-        Optional<Snowflake?> parentId = default,
+        Optional<Snowflake?> parentID = default,
         Optional<VideoQualityMode?> videoQualityMode = default,
         Optional<bool> isArchived = default,
         Optional<AutoArchiveDuration> autoArchiveDuration = default,
         Optional<bool> isLocked = default,
+        Optional<bool> isInvitable = default,
```

Comment:
Remember to add this to ModifyThreadChannelAsync below.

Reply:
Good catch, will add it later